src/client/ClientGameEngine.cpp

```diff
@@ -15,345 +15,339 @@
-                    }
-
-                    time_t modified = file_stat.st_mtime;
-                    lastUpdateTimer = clientGame.getCurrentTime();
-
-                    if (modified != last_modified)
-                    {
-                        last_modified = modified;
-                        clientGame.updateElementTexture();
-                        turn++;
-                    }
+                if (modified != last_modified)
+                {
+                    last_modified = modified;
+                    clientGame.updateElementTexture();
+                    turn++;
                 }
             }
         }
-        else
-        {
-            lockIf.unlock();
-        }
     }
+    else
+    {
+        lockIf.unlock();
+    }
+}
 
-    /*!
-    * @brief Loop that is used when the client is in MenuMode
-    */
-    void ClientGameEngine::playMenu()
+/*!
+ * @brief Loop that is used when the client is in MenuMode
+ */
+void ClientGameEngine::playMenu()
+{
+    std::unique_lock<std::mutex> lockIf(mutexRunningEngine);
+    if (runningWindow == MENU)
     {
-        std::unique_lock<std::mutex> lockIf(mutexRunningEngine);
-        if (runningWindow == MENU)
-        {
-            lockIf.unlock();
+        lockIf.unlock();
 
-            std::thread t(&ClientGameEngine::startMenuWindow, this);
+        std::thread t(&ClientGameEngine::startMenuWindow, this);
 
-            while (true)
-            {
+        while (true)
+        {
 
-                std::unique_lock<std::mutex> lockWhile(mutexRunningEngine);
-                if (runningWindow != MENU)
-                {
-                    lockWhile.unlock();
-                    t.join();
-                    break;
-                }
+            std::unique_lock<std::mutex> lockWhile(mutexRunningEngine);
+            if (runningWindow != MENU)
+            {
                 lockWhile.unlock();
+                t.join();
+                break;
             }
-        }
-        else
-        {
-            lockIf.unlock();
+            lockWhile.unlock();
         }
     }
-
-    /*!
-    * @brief Detect an intersection between a point and a rect
-    * @param point the point
-    * @param rectangle the rectangle
-    */
-    bool ClientGameEngine::intersectPointRect(sf::Vector2i point, sf::FloatRect rectangle)
+    else
     {
-        return (   point.x >= rectangle.left 
-                && point.x <= rectangle.left + rectangle.width 
-                && point.y >= rectangle.top 
-                && point.y <= rectangle.top + rectangle.height);
+        lockIf.unlock();
     }
+}
 
+/*!
+ * @brief Detect an intersection between a point and a rect
+ * @param point the point
+ * @param rectangle the rectangle
+ */
+bool ClientGameEngine::intersectPointRect(sf::Vector2i point, sf::FloatRect rectangle)
+{
+    return (point.x >= rectangle.left && point.x <= rectangle.left + rectangle.width && point.y >= rectangle.top && point.y <= rectangle.top + rectangle.height);
```

Comment:
Better before with 4 line

Reply:
`    return (point.x >= rectangle.left &&
            point.x <= rectangle.left + rectangle.width &&
            point.y >= rectangle.top &&
            point.y <= rectangle.top + rectangle.height);`
Like that with the && at the end you can do ctrl shift i and it stay like that